src/dymaptic.GeoBlazor.Core/Components/Views/MapView.razor.cs

```diff
@@ -24,6 +31,11 @@ namespace dymaptic.GeoBlazor.Core.Components.Views;
 /// </example>
 public partial class MapView : MapComponent
 {
+    /// <summary>
+    /// Event fires allways when view has rendered
+    /// </summary>
+    public event AsyncEventHandler? ViewRendered;
```

Comment:
Assuming the point of this event is for internal components to receive notification, I think we could make this `internal` instead of public, to avoid confusion for consumers with `OnMapRenderedHandler`. We could also name it something special like `ViewRenderedInternalEvent`. Also, curious on your thoughts if my `OnMaprenderedHandler` and my PR's `OnMapRendered` EventCallback should actually be `OnViewRendered`? Since views and maps are 1-1, I tend to conflate them more than I should.